filter_plugins/filter_hosts.py

```diff
@@ -87,6 +94,30 @@ def get_instance_systemd_service(app_name, inventory_hostname, stateboard):
     return '{}@{}'.format(app_name, inventory_hostname)
 
 
+def get_tasks_dict_from_dir(dir_path):
+    tasks = {}
+
+    if dir_path is None:
+        return tasks
+
+    files = os.listdir(dir_path)
+    files = filter(lambda x: x.endswith('.yml'), files)
+    for file in files:
+        name = os.path.splitext(os.path.basename(file))[0]
+        tasks[name] = os.path.abspath(os.path.join(dir_path, file))
+
+    return tasks
```

Comment:
`tasks` -> `tasks_paths`? `paths_by_tasks`?

`tasks` looks like a list of tasks.